pkg/plugin/connector/service.go

```diff
@@ -57,24 +69,29 @@ func NewPluginService(
 	}
 }
 
+func (s *PluginService) Init(ctx context.Context, connUtilsAddr string, connUtilsToken string) {
+	s.builtinReg.Init(ctx)
+	s.standaloneReg.Init(ctx, connUtilsAddr, connUtilsToken)
+}
+
 func (s *PluginService) Check(context.Context) error {
 	return nil
 }
 
-func (s *PluginService) NewDispenser(logger log.CtxLogger, name string) (Dispenser, error) {
+func (s *PluginService) NewDispenser(logger log.CtxLogger, name string, connectorID string) (Dispenser, error) {
```

Comment:
Seems like in this function we have all the required information to create `pconnector.PluginConfig` - in the next PR I propose managing the config and token here (create, register so it can be used in conn utils, deregister when connector stops), otherwise we'll do the same work twice in the builtin and standalone dispenser.

Reply:
Yup, makes sense!